mlflow/environment_variables.py

```diff
@@ -0,0 +1,12 @@
+"""
+This module defines environment variable names used in MLflow.
+"""
+
+#: Specify maximum retries for mlflow rest API http request.
+MLFLOW_HTTP_REQUEST_MAX_RETRIES = "MLFLOW_HTTP_REQUEST_MAX_RETRIES"
+
+#: Specify backoff factor for mlflow rest API http request.
```

Comment:
```suggestion
#: Specify backoff factor for mlflow http request.
```

Do we need `rest API`?